leafsmart-front-end/src/components/ForecastWeather.js

```diff
@@ -0,0 +1,69 @@
+import { React, useState, useEffect } from "react";
+
+import axios from "axios";
+import "./CurrentWeather.css";
+import ForecastWeatherCard from "./ForecastWeatherCard";
+import { forecastDays } from "../helpers/selectors";
+
+const ForecastWeather = (props) => {
+  const { coordinates } = props;
+  const [forecastWeather, setForecastWeather] = useState([]);
+  const [isLoading, setIsLoading] = useState([false]);
+
+  const apiUrl = process.env.REACT_APP_WEATHERBIT_ENDPOINT_ENDPOINT;
+  const apiKey = process.env.REACT_APP_WEATHERBIT_ENDPOINT_KEY;
+
+  useEffect(() => {
+    if (coordinates.length > 0) {
+      console.log(coordinates);
+      const [lat, lon] = coordinates;
+
+      // Weatherbit API Endpoint
+      setIsLoading(true);
+      const forecastWeatherEndpoint = `${apiUrl}/forecast/daily?&lat=${lat}&lon=${lon}&key=${apiKey}`;
+
+      axios
+        .get(forecastWeatherEndpoint)
+        .then((res) => {
+          const data = res.data["data"];
+          setForecastWeather(data);
+          setIsLoading(false);
+          //console.log(data);
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+    }
+  }, [coordinates, apiUrl, apiKey]);
+
+  const forecastDaysArr = forecastWeather
+    .splice(1, 6)
+    .map((forecastWeather, index) => {
+      return (
+        <ForecastWeatherCard
+          key={forecastWeather.id}
+          icon={forecastWeather.weather.icon}
+          description={forecastWeather.weather.description}
+          temp={forecastWeather.temp}
+          day={forecastDays()[index]}
+        />
+      );
+    });
+
+  return (
+    <main>
+      <section>
+        {isLoading ? (
```

Comment:
This line's going to cause isLoading to show before a person does their search -- do we want that, or do we want to quickly fix that before merging?

Reply:
yes let's change this!

Reply:
Awesome, thanks!